config/routes.rb

```diff
@@ -19,7 +19,12 @@
 
   get 'workflows/:workflow_id/extractors/:extractor_id/extracts', to: 'extracts#index'
   put 'workflows/:workflow_id/extractors/:extractor_id/extracts', to: 'extracts#update'
+  get 'workflows/:workflow_id/extractors/export', to: 'data_requests#request_extracts'
 
   get 'workflows/:workflow_id/reducers/:reducer_id/reductions', to: 'reductions#index'
   put 'workflows/:workflow_id/reducers/:reducer_id/reductions', to: 'reductions#update'
+  get 'workflows/:workflow_id/reducers/export', to: 'data_requests#request_reductions'
+
+  get 'data_request/:request_id', to: 'data_requests#check_status'
```

Comment:
Oh, it doesn't need to be that much work to change these routes. I'll see if I can make some time to send you a patch to the PR tomorrow.